<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsScripting</class>
 <widget class="QWidget" name="PrefsScripting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scripting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="EnableScripts">
     <property name="text">
      <string>&amp;Enable scripting</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_world_script">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>World script file</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="WorldScript">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="WorldScript_browse">
          <property name="text">
           <string>Browse...</string>
          </property>
          <property name="icon">
           <iconset theme="document-open"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="WorldScript_create">
          <property name="text">
           <string>Create</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="WorldScript_edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="fileNotFound">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="visible">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>File not found</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="ScriptReloadOption_label">
          <property name="text">
           <string>Recompile when script file changes:</string>
          </property>
          <property name="buddy">
           <cstring>ScriptReloadOption</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ScriptReloadOption">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Confirm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Always</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Never</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_output">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="NoteTextColour_label">
        <property name="text">
         <string>Display notes in</string>
        </property>
        <property name="buddy">
         <cstring>NoteTextColour</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ColorPickerButton" name="NoteTextColour">
        <property name="baseSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="ColorPickerButton" name="NoteBackgroundColour">
        <property name="baseSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="NoteBackgroundColour_label">
        <property name="text">
         <string>on</string>
        </property>
        <property name="buddy">
         <cstring>NoteBackgroundColour</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ErrorTextColour_label">
        <property name="text">
         <string>Display errors in</string>
        </property>
        <property name="buddy">
         <cstring>ErrorTextColour</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ColorPickerButton" name="ErrorTextColour">
        <property name="baseSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="ErrorBackgroundColour_label">
        <property name="text">
         <string>on</string>
        </property>
        <property name="buddy">
         <cstring>ErrorBackgroundColour</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="ColorPickerButton" name="ErrorBackgroundColour">
        <property name="baseSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerButton</class>
   <extends>QToolButton</extends>
   <header>colorpickerbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
